coralproject/talk
# [CORL-1749] Live Chat: stop navigation buttons from blocking other stream controls

## What does this PR do?

Forces flex alignment `div`'s in the live chat navigation buttons from having a height so they don't block click events behind them on the stream.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to live chat
- View stream when not tailing
- Post a comment with a different account + user
- See the "new comments" or "jump to live" show up
- Line up the button with the "rec", "reply", and "show conversation" buttons of a comment
- Click on the "rec", "reply", or "show conversation" and it will work, no longer blocked by the invisible Flex div that aligns the button center at the bottom of the stream
